### ISSUE TITLE
Update the progress when sync

### ISSUE THREAD

**Issue by cauealves** [pgherveou/gulp-awspublish#96]
Hello,

When my credentials do not have permissions for delete a object, the `sync` are showing the object are deleted but is wrong.

**Comment by pgherveou** [comment]
Yes this is because we are deleting all the files in bulk when the stream ends
https://github.com/pgherveou/gulp-awspublish/blob/master/lib/index.js#L390
We dont handle the error, I guess we should report it somehow to the user.
Feel free to send a PR if you want to implement that.
